uv.lock

```diff
@@ -1,5 +1,4 @@
 version = 1
```

Comment:
I say merge this commit with another chore commit or if there isn't one, merge it with a normal commit that it most related. We don't need to have a separate commit for uv lock updates as the commit doesn't achieve any goal by itself.